examples/Wiper/Wiper.ino

```diff
@@ -0,0 +1,39 @@
+/*
+ Using a servo as a car Wiper, activated by switch.
+
+ modified on 19 March 2020
+ by Durgesh Pachghare
+*/
+
+#include <Servo.h>
+
+Servo myservo;  // create servo object to control a servo
+
+int button_pin = 2;  // digital pin used to connect the button input
+int val;    // variable to read the value from the analog pin
+int startpos = 10;      //angle from which the servo wiper starts
+int endpos = 145;       //angle till the servo wiper rotates
+
+void setup() 
+{
+  myservo.attach(9);  // attaches the servo on pin 9 to the servo object
+  pinMode(button_pin,INPUT);    //declares the pin as digital input
+  myservo.write(startpos);      // Initialize the servo wiper to start angle
+}
+
+void loop() 
+{
+    if(digitalRead(button_pin))     //Start the servo if the button is pressed and complete the entire rotation regardless of button state later

```

Comment:
```suggestion
    if(digitalRead(button_pin) == HIGH)     //Start the servo if the button is pressed and complete the entire rotation regardless of button state later

```
This makes things more clear to the user. It's also not really best practices to assume the values of `HIGH` and `LOW`. Sure, `HIGH` happens to be defined as 1 and `LOW` as 0, but there is nothing that guarantees this will always be the case.

Reply:
Thank you so much, I have added that to LOW since we have added INPUT_PULLUP now. I also added a necessary description.